security/pkg/util/configutil.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2020 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+import (
+	"fmt"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	corev1 "k8s.io/client-go/kubernetes/typed/core/v1"
+
+	"istio.io/pkg/log"
+)
+
+// InsertDataToConfigMap inserts a data to a configmap in a namespace.
+// client: the k8s client interface.
+// namespace: the namespace of the configmap.
+// value: the value of the data to insert.
+// configName: the name of the configmap.
+// dataName: the name of the data in the configmap.
+func InsertDataToConfigMap(client corev1.CoreV1Interface, namespace, value, configName, dataName string) error {
+	configmap, err := client.ConfigMaps(namespace).Get(configName, metav1.GetOptions{})
```

Comment:
I am pretty sure you can just do .Update() and it will create it for you if needed. Should verify though

Reply:
Done.

Reply:
After changing to just do Update(), the unit tests failed with error: error when updating configmap test-configmap-name: configmaps "test-configmap-name" not found.
So to update, the configmap needs to exist.